StarkInfra/StarkInfra/IndividualDocument/IndividualDocument.cs

```diff
@@ -0,0 +1,331 @@
+        ///     <item>before [DateTime, default null] date filter for objects created only before specified date. ex: DateTime(2020, 3, 10)</item>
+        ///     <item>status [list of string, default null]: filter for status of retrieved objects. ex: "active", "blocked", "expired" or "canceled"</item>
+        ///     <item>tags [list of strings, default null]: list of tags to filter retrieved objects. ex: new List<string>{ "tony", "stark" }</item>
+        ///     <item>ids [list of strings, default null]: list of ids to filter retrieved objects. ex: new List<string>{ "5656565656565656", "4545454545454545" }</item>
+        ///     <item>user [Organization/Project object, default null]: Organization or Project object. Not necessary if StarkInfra.Settings.User was set before function call</item>
+        /// </list>
+        /// <br/>
+        /// Return:
+        /// <list>
+        ///     <item>IEnumerable of IndividualDocument objects with updated attributes</item>
+        /// </list>
+        /// </summary>
+        public static IEnumerable<IndividualDocument> Query(int? limit = null, DateTime? after = null, DateTime? before = null, 
+            List<string> status = null, List<string> tags = null, List<string> ids = null, User user = null
+        )
+        {
+            (string resourceName, Api.ResourceMaker resourceMaker) = Resource();
+            return Rest.GetList(
+                resourceName: resourceName,
+                resourceMaker: resourceMaker,
+                query: new Dictionary<string, object> {
+                    { "limit", limit },
+                    { "after", after },
+                    { "before", before },
+                    { "status", status },
+                    { "tags", tags },
+                    { "ids", ids }
+                },
+                user: user
+            ).Cast<IndividualDocument>();
+        }
+
+        /// <summary>
+        /// Retrieve paged IndividualDocument objects
+        /// <br/>
+        /// Receive a list of up to 100 IndividualDocument objects previously created in the Stark Infra API and the cursor to the next page.
+        /// Use this function instead of query if you want to manually page your requests.
+        /// <br/>
+        /// Parameters (optional):
+        /// <list>
+        ///     <item>cursor [string, default null]: cursor returned on the previous page function call</item>
+        ///     <item>limit [integer, default 100]: maximum number of objects to be retrieved. Max = 100. ex: 35.</item>
+        ///     <item>after [DateTime, default null] date filter for objects created only after specified date. ex: DateTime(2020, 3, 10)</item>
+        ///     <item>before [DateTime, default null] date filter for objects created only before specified date. ex: DateTime(2020, 3, 10)</item>
+        ///     <item>status [list of string, default null]: filter for status of retrieved objects. ex: "active", "blocked", "expired" or "canceled"</item>
```

Comment:
- status [list of strings, default None]: filter for status of retrieved objects. Options: ["created", "canceled", "processing", "failed", "success"]